private/buf/cmd/buf/command/beta/registry/commit/commitlist/commitlist.go

```diff
@@ -42,8 +42,7 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		//Use:   name + " <buf.build/owner/repo:branch>",
-		Use:   name + " <buf.build/owner/repo>",
+		Use:   name + " <buf.build/owner/repo[:track]>",
```

Comment:
Does this technically support tags? I know it would only ever list a single commit, but we might want this to be `[:reference]` if tags are supported by the `ListRepositoryCommitsByReference` endoint.

Reply:
Yes it does technically support tags (also commit names). I thought about making this [:ref] to match `... commit get`, but went with track because that's the only one where it makes sense to use list instead of get. On second thought I think I do like [:ref] better. Let users decide what makes more sense to use.